docs/shared-content/octopus-recommendations/project-recommendations.include.md

```diff
@@ -3,32 +3,71 @@ We built Octopus Deploy with the core concept of consistency across all environm
+The workflow would be:
+1. Create a branch, commit some changes.
+2. Build is triggered on branch check-in.  It creates a release in Octopus for the Development lifecycle and pushes to Dev.
+3. Changes are verified in Dev and are promoted to QA.
+4. Full test suite is run in QA.
+5. Bugs or changes are found, repeat the previous steps.
+6. After a few iterations the change is ready for Production.
+7. Create a pull request and merge into main.
+8. Build is triggered on check-in to main.  It creates a release in Octopus for the Release lifecycle and pushes to Staging.
+9. Automated tests are run in Staging.
+10. Assuming tests pass, promote to Production.  If tests don't pass then a new branch is created and this process starts all over.
 
-Orchestration allows you to isolate your code in your source control repository and have separate builds for each component.  This way, your CI/CD pipeline only has to build and deploy something that changed rather than the entire application.  An added benefit is reduced build and deployment times.  
+Octopus Deploy provides the capability for dynamic package / docker image selection.  Meaning, you can have a different package per environment.  The intended use case for this is when using a third-party external feed and feed changes between environments.  The external feed provides the capabilities to "promote" packages ready for deployment.
+
+We do not recommend having a single lifecycle with all environments.  When that happens, we will see customers create a single release, and then change the package or package version when going from QA to Staging.  Such an approach is very hard to audit and track.  
+
+Changes made on feature or short-lived branches are not ready for Production.  They should be deployed to testing environments for verification and testing, but they should never have the chance to make it to Production.  Merging into main should trigger a fresh build because you could be merging multiple changes from different branches for the first time.  The underlying code has changed and a new build is needed to test and verify.
+
+For the packages / docker containers built off of branches, append a pre-release tag to the release version.  Leverage channel version rules to only allow packages / docker containers with a pre-release tag for the Development lifecycle.  At the same time, only allow packages / docker containers WITHOUT a pre-release tag for the release lifecycle.
+
+:::hint
+This section is another reason why we recommend deploying all tightly coupled components stored in the same source control repository within the same project.  Attempting to coordinate different lifecycles and releases across multiple projects can add aditional overhead which runs the risk of something not working correctly.
+:::
```

Comment:
```suggestion
Octopus Deploy provides the capability for dynamic package / docker image selection.  This allows you to have a different package per environment.  The intended use case is when using a third-party external feed and the feed changes between environments.  The external feed provides the capabilities to "promote" packages ready for deployment.

We don't recommend having a single lifecycle with all environments.  When that happens, we have seen customers create a single release and change the package or package version from QA to Staging.  Such an approach is challenging to audit and track.  

Changes made on feature or short-lived branches are not ready for Production.  They should be deployed to testing environments for verification and testing, but they should never have the chance to make it to Production.  Merging into main should trigger a fresh build because you could be merging multiple changes from different branches for the first time.  The underlying code has changed, and a new build is needed to test and verify.

For the packages / docker containers built from branches, append a pre-release tag to the release version.  Leverage channel version rules to only allow packages / docker containers with a pre-release tag for the Development lifecycle.  At the same time, only allow packages / docker containers **without** a pre-release tag for the release lifecycle.

:::hint
This section is another reason we recommend deploying all tightly coupled components stored in the same source control repository within the same project.  Attempting to coordinate different lifecycles and releases across multiple projects can add aditional overhead, which runs the risk of something needing to be fixed.
:::
```